assets/src/components/Digitizing.js

```diff
@@ -72,8 +79,16 @@ export default class Digitizing extends HTMLElement {
 
         const mainTemplate = () => html`
         <div class="digitizing">
-            <div class="digitizing-buttons btn-group dropend" data-bs-toggle="tooltip" data-bs-title="${lizDict['digitizing.toolbar.drawTools']}">
-                <button type="button" class="btn ${this.deactivate ? '' : 'active btn-primary'}" @click=${(event) => {this.toggleToolSelected(event)}}>
+            <div
+                    class="digitizing-buttons btn-group dropend"
+                    data-bs-toggle="tooltip"
+                    data-bs-title="${lizDict['digitizing.toolbar.drawTools']}"
+            >
```

Comment:
```suggestion
            <div
                class="digitizing-buttons btn-group dropend"
                data-bs-toggle="tooltip"
                data-bs-title="${lizDict['digitizing.toolbar.drawTools']}"
                >
```
I am more in favor of this formatting.